### DISCUSSION TITLE
Can I make a formula with Java?

### DISCUSSION THREAD

**Discussion by pacman-admin** [Homebrew/discussions#4979] [Writing Formulae/Casks]
### Output of `brew config`

```shell
Does it matter?
```

### Output of `brew doctor`

```shell
Is this relevant?
```

### Description of issue

I want to be able to write a formula in java.
Java is easy for me (it like 2 years to learn).
I also know Python but would like to take advantage of Java's performance advantages(Bonus!).

**Comment by SMillerDev** [comment]
You can write a formula for anything. A formula file is a specific ruby DSL though.
